pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -187,14 +187,12 @@ func (r *ReconcileNodeMaintenance) Reconcile(request reconcile.Request) (reconci
 	}
 
 	// Cordon node
```

Comment:
Maybe move this comment to line 194?

Reply:
I moved it intentionally from 194 above to mark the entire process of uncordon - adding or removing the taints is basically uncordoning/cordoning the node